#!/usr/bin/env python
PACKAGE = "camera_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("distance_detection", double_t, 0, "distance de detection", 50., 20., 100.)

exit(gen.generate(PACKAGE, "depth_analysis", "depth_analysis"))
